clang/test/Sema/return-type.c

```diff
@@ -0,0 +1,29 @@
+// RUN: %clang_cc1 -Wreturn-type -Wno-return-mismatch -fsyntax-only -verify %s
```

Comment:
Because this is the same test code in both files, let's remove this file, add this RUN line to the other file, and rename the other file `return-type-mismatch.h`.

Reply:
done